### ISSUE TITLE
Not working on Raspbian

### ISSUE THREAD

**Issue by AymanFadda** [Steveorevo/node-red-contrib-nbrowser#14]
Hi,
I've been trying for days to make it work but I get no answer at all. I tried even to follow the example but the reported site isn't even working.
Is it possible to make an easy example that shows how it should work?

Thank you

**Comment by Steveorevo** [comment]
Are you getting a particular error message? Are you using a window manager, or command line (the latter requires xvfb).

**Comment by AymanFadda** [comment]
I have no error at all. I just don't receive anything. It seems it's not working at all.
For what is concerning the xvfb, since I have not too much experience, would you please explain how to install it?
Thanks

**Comment by Steveorevo** [comment]
Hi Aynan, I’m sorry but if you don’t have anything at all; I’m not able to figure out where to begin; without a screenshot, text from a command line, I’m not able to know where/what you are coming from.

**Comment by meeki007** [comment]
I too have this problem. From what i can tell its an issue with raspbian-stretch.

AymanFadda did you install it from GUI and not the command line? I did the same the first time and had the same problems as you. I rebooted node-red as stated on the install page.
I get no errors.... I get nothing.....It just hangs on what ever page i try to load.

So I went to terminal on a fresh pi running node-red that never had nbrowser installed.

Her is what i figured out:

nightmare": "^2.10.0" requires a higher version of electron than runs on arm.
electron@1.7.9 is the only one working right now.

so even if you get a working electron and Xvfb working it will still try to install a newer version of electron.

Its not the Steveorevo's fault. Its whoever is building the package for electron.

So I went oldschool and now have phantomjs and casperjs running on the PI it was a pretty EZ install.
I'm now looking into Steverevo's code and thinking about modding the hell out of it to make it work with casperjs.

Meeki
